.github/pull_request_template.md

```diff
@@ -0,0 +1,84 @@
+-->
+
+
+### What Should Reviewers Watch For?
+
+If you're reviewing this PR, please check these things, in particular:
+
+* TODO
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
+
+
+### What Security Implications Does This PR Have?
+
+Submitters should complete the following questionnaire:
+
+* If the answer to any of the questions below is **Yes**, then here's a link to the associated Security Impact Assessment: N/A.
+    * Does this PR add any new software dependencies? **Yes** or **No**.
+    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
+    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
+* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
+    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.
+
+
+### What Needs to Be Deployed Before this PR?
+
+This PR cannot be deployed until the following pre-requisite changes have been fully deployed:
+
+* CMSgov/some_repo#42
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply.
+
+Common items include:
+* Database migrations (which should always go out in PRs by themselves, to reduce risk).
+* New features in external dependencies (e.g. BFD).
+-->
+
+
+### Submitter Checklist
+
+Before requesting final review, I've gone through and verified that this PR complies with the following requirements:
+
+* [ ] I've verified that this PR is as small as it can reasonably be, in order to:
+    1. Ensure that any problems it causes have a small "blast radius".
+    2. Ensure that it'll be easier to rollback if that becomes necessary.
+    3. Ease the burden on reviewers.
+* [ ] I've verified that the individual commits in this PR all:
+    1. Reasonably explain the "what" and "why" of the changes.
+    2. Leverage JIRA's [smart commits](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) to reference the JIRA ticket that they're associated with.
+* [ ] I've verified that this PR includes all required documentation changes, including changelog / release notes entries.
+* [ ] I've verified that all new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers unfamiliar with the project.
```

Comment:
There will be code that simply will not be reasonably clear to engineers unfamiliar with the project, typically because the code requires significant domain knowledge or significant understanding of an uncommon framework. How about `reasonably clear to engineers, preferably ones unfamiliar with the project.`